src/ops/cloud/SecretsOps.ts

```diff
@@ -372,6 +438,16 @@ export async function disableVersionOfSecret({
   });
 }
 
+export async function readSecret({
+  secretId,
+  state,
+}: {
+  secretId: string;
+  state: State;
+}): Promise<SecretSkeleton> {
+  return await _getSecret({ secretId, state });
+}
+
```

Comment:
This is just wrapping _getSecret anyway, so we can probably just continue to re-export the function like they were doing it before on.